main.py

```diff
@@ -2,6 +2,55 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
```

Comment:
It's better to avoid using Django ORM in the main function. It's a good practice to separate business logic from data access logic. Consider moving the ORM queries to separate functions in the `db` module.